src/metrics/definitions.md

```diff
@@ -0,0 +1,9 @@
+# Metric Definitions
```

Comment:
This all smells very GUD-centric: it's describing GUD's method for specifying metrics. Does a DTMO page titled "Metric Definitions" need to be more general than this, or are we OK with neglecting metrics that don't fit this format?

Reply:
Good point.  Part of that is due to some copy and paste history.  I'll revise.  Generally, the GUD metrics are an important class and we definitely want to support first-class concepts of usage criteria, but we should make clear that not all metrics require usage criteria.  Sound fair?

Reply:
Is this breakdown into "usage criterion", "dimension" and "metric" useful or confusing outside of a GUD context, even if parts are marked as optional?

This breakdown works well for metrics that boil down to "count the users who meet this usage criterion over a time period defined w.r.t. this day, optionally splitting users by some dimension(s)". This is one of my favourite forms of metric: "set some criteria and count users" is very outlier-resistant, and it encourages people to think and in terms of individual users and their experiences, rather than encouraging people to design a car with room for 2.1 kids when the vast majority of families have an integer number of children.

But this breakdown has a narrow scope that excludes lots of metrics, for example, "Average URIs browsed per client per day", "Total searches made per day", or any funnel conversion rate metrics. I know that GUD supports parameterised retention metrics that compute fractions, but even _that_ doesn't fit this breakdown as described.

What value are we trying to get from this document? How can we get that value in a way that doesn't appear to limit us to a subset of the metrics we need to care about?

Reply:
Discussed in thread above.